tests/functional/yarn.lock

```diff
@@ -264,6 +264,28 @@
   resolved "https://registry.yarnpkg.com/@discoveryjs/json-ext/-/json-ext-0.5.7.tgz#1d572bfbbe14b7704e0ba0f39b74815b84870d70"
   integrity sha512-dBVuXR082gk3jsFp7Rd/JI4kytwGHecnCoTtXFb7DB6CNHp4rg5k1bhg0nWdLGLnOV71lmDzGQaLMy8iPLY0pw==
 
+"@heartexlabs/ls-test@git+ssh://git@github.com/heartexlabs/ls-frontend-test#36fc7ffc2cddc4e77aa785ec3304900b27163f72":
```

Comment:
replace the next occurency, I don't know why did it left it intact

Reply:
Somehow it works fine with that. I mean sometimes `yarn add` keeps it and sometimes removes it but it still works so I don't think we need to remove it manually every time.

Reply:
I just wanted to highlight that if yarn puts it here most probably it's because of some local leftovers. But it really affects nothing, so okay